influxql/iterator.gen.go.tmpl

```diff
@@ -657,6 +691,7 @@ func (itr *{{$k.name}}InterruptIterator) Next() *{{$k.Name}}Point {
 type {{$k.name}}AuxIterator struct {
 	input      *buf{{$k.Name}}Iterator
 	output     chan *{{$k.Name}}Point
+	err        chan error
 	fields     auxIteratorFields
 	background bool
 }
```

Comment:
It seems like using a `chan struct{ *{{$k.Name}}Point, error }` would more closely match the signature of `Next()`. I'm worried that two channels could cause timing issues in the future and they may also add additional lock overhead.